fe/fe-core/src/test/java/com/starrocks/sql/plan/PipelineParallelismTest.java

```diff
@@ -72,22 +71,6 @@ public void testMetaScan() throws Exception {
         Assert.assertEquals(1, fragment1.getPipelineDop());
     }
 
-    @Test
-    public void testOutfile() throws Exception {
-        ExecPlan plan = getExecPlan("SELECT v1,v2,v3 FROM t0  INTO OUTFILE \"hdfs://path/to/result_\""
-                + "FORMAT AS CSV PROPERTIES" +
-                "(\"broker.name\" = \"my_broker\"," +
-                "\"broker.hadoop.security.authentication\" = \"kerberos\"," +
-                "\"line_delimiter\" = \"\n\", \"max_file_size\" = \"100MB\");");
-        System.out.println(plan.getExplainString(StatementBase.ExplainLevel.COST));
-        PlanFragment fragment0 = plan.getFragments().get(0);
-        assertContains(fragment0.getExplainString(TExplainLevel.NORMAL), "RESULT SINK");
-        // Outfile ResultSink doesn't support pipeline, so ParallelExecNum of fragment is 
-        // equal to the corresponding session variables.
-        Assert.assertEquals(parallelExecInstanceNum, fragment0.getParallelExecNum());
-        Assert.assertEquals(1, fragment0.getPipelineDop());
```

Comment:
We could retain this test case to check whether all the things are OK when outfile is executed on pipeline.
```
        Assert.assertEquals(1, fragment0.getParallelExecNum());
        Assert.assertEquals(1, fragment0.getPipelineDop());
```